app/src/main/res/layout/fragment_search.xml

```diff
@@ -15,23 +15,34 @@
   ~ limitations under the License.
   -->
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-             android:layout_width="match_parent"
-             android:layout_height="match_parent">
+<RelativeLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+    <ProgressBar
+            style="@android:style/Widget.ProgressBar.Horizontal"
+            android:id="@+id/progressBar_searchProgress"
```

Comment:
Keep the `android:id` first. Then apply common View attributes, then attributes only applicable to this ProgressBar. Do this generally in this layout.